### DISCUSSION TITLE
Question about blaze and sponsor post

### DISCUSSION THREAD

**Discussion by LadyHaywood** [AprilSylph/XKit-Rewritten#826] [Help Desk]
Am curious to know what  anti capitalism is doing to hide blaze and sponsor ads that act like posts and if it's something I can put into blokada so I can hide those types of post on mobile. Is that something I can actually do?

**Comment by marcustyphoon** [comment]
I didn't look into it much to see exactly what it can do, but if Blokada is only able to block network requests, it won't be able to do this, no.

Being a web extension allows XKit Rewritten to hide individual UI elements in a web browser; this can't be done for an iOS or Android app at all (that would be hacking the app - would be neat if you could, though). In a mobile browser, you would use a web extension to do this... i.e. XKit Rewritten, which can run on mobile web on Android in e.g. Kiwi Browser, and can technically be run in iOS Safari if you're a developer.

Not familiar with Blokada, but it definitely can't hack the app and I assume(?) it doesn't have a web extension component that interfaces with mobile browsers.
